addons/mail/static/src/models/composer/composer.js

```diff
@@ -311,6 +311,9 @@ function factory(dependencies) {
             if (!this.thread) {
                 return;
             }
+            if (this.suggestionModelName === 'mail.channel_command') {
+                return;
+            }
```

Comment:
Actually my first suggestion of using `_getCommandFromText` must still be applied too.
Because some commands require more content eg. `/lead lead title`

`this.suggestionModelName === 'mail.channel_command'` should cover the case where a command is in progress

And `_getCommandFromText` should cover the case where the command was fully selected and suggestion closed


Also please add a test to cover the fix.

Reply:
Hello, this is still needed, when typing `/lead lead title` it is currently sending typing but it should not.